README.md

```diff
@@ -163,6 +163,13 @@ The `commentstart` linter can automatically fix comments that do not start with
 
 When the `json` tag is present, and matches the first word of the field comment in all but casing, the linter will suggest that the comment be updated to match the `json` tag.
 
+## DuplicateMarkers
+
+The `duplicatemarkers` linter checks for duplicate markers related with specific type or field.
+This linter diagnose _only_ if the marker and value together are completely unique. e.g. the `duplicatemarkers` will not diagnose the field has kubebuilder:validation:MaxLength=10 and kubebuilder:validation:MaxLength=11
+
+About the duplicated markers which has different value, it requires specific rule for each marker, these are processed by its corresponding linter.
+
```

Comment:
I thought we agreed we would add something into this linter, but at a later date? Perhaps we don't want to add this bit of documentation for now?